test.py

```diff
@@ -0,0 +1,18 @@
+from viewer import PdfViewer
+
+
+def main(): 
+    #pdfPath = 'examples/Example-1.pdf'
+
+    print("Please enter the path to your PDF file: ")
+    pdfPath = input()
+    test = PdfViewer(pdfPath)
+    print(test.pdf.get_num_pages())
+
+    print("Which page should be displayed? (Start at 0): ")
+    index = input()
```

Comment:
If we are taking user input for the page number, make sure to convert the input into int. By default, Python takes the input as a string so we need to convert it.